egov-indexer/pgr-migration-batch-indexer.yml

```
@@ -0,0 +1,65 @@
+                    outJsonPath: $.Data.slaHours
+              - path: http://egov-mdms-service.egov:8080/egov-mdms-service/v1/_search
+                moduleName: tenant
+                masterName: tenants
+                tenantId: pg
+                filter: "[?(@.code == $tenant)]"
+                filterMapping:
+                  - variable: $tenant
+                    valueJsonpath: $.service.tenantId
+                uriResponseMapping:
+                  - inJsonPath: $.MdmsRes.tenant.tenants
+                    outJsonPath: $.Data.tenantData
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)